common/sgx/tcbinfo.c

```diff
@@ -326,6 +329,10 @@ const char* oe_sgx_tcb_status_str(const oe_sgx_tcb_status_t sgx_tcb_status)
             return SGX_TCB_STATUS_SW_HARDENING_NEEDED;
         case OE_SGX_TCB_STATUS_CONFIGURATION_AND_SW_HARDENING_NEEDED:
             return SGX_TCB_STATUS_CONFIGURATION_AND_SW_HARDENING_NEEDED;
+        case OE_SGX_TCB_STATUS_TD_RELAUNCH_ADVISED:
```

Comment:
We also need to handle the new error codes so the attestation does not fail
https://github.com/openenclave/openenclave/blob/ca67307dc46ebcdcc38b48ec121fc2168d3638cd/host/sgx/sgxquote.c#L1605C27-L1605C54

Reply:
Should we set the result to `OE_TCB_LEVEL_INVALID` if we encounter the new error code?

Reply:
What's the new error code?

Reply:
Checked with Mingwei offline, and we agree that no additional handling of TD RELAUNCH ADVISED result since it won't fail the `oe_tdx_verify_quote` call.